api/python/tests/test_workflows.py

```diff
@@ -212,26 +212,29 @@ def test_validate_pkg_entries_fail(self, get_pkg_entries_for_validation_mock):
 class GetPkgEntriesForValidationTest(QuiltTestCase, WorkflowValidatorTestMixin):
     def test(self):
         entries_data = {
-            'b/a': bytes(1),
-            'a/b': bytes(2),
-            'c': bytes(3),
+            'b/a': (bytes(1), None),
+            'a/b': (bytes(2), {}),
+            'c': (bytes(3), {"test": "test"}),
         }
         pkg = Package()
-        for lk, data in entries_data.items():
-            pkg.set(lk, data)
+        for lk, (data, meta) in entries_data.items():
+            pkg.set(lk, data, meta=meta)
 
         workflow_validator = self.get_workflow_validator()
         assert workflow_validator.get_pkg_entries_for_validation(pkg) == [
             {
                 'logical_key': 'a/b',
                 'size': 2,
+                "meta": {},
             },
             {
                 'logical_key': 'b/a',
                 'size': 1,
+                "meta": {},
             },
             {
                 'logical_key': 'c',
                 'size': 3,
+                "meta": {"test": "test"},
```

Comment:
`"meta": {"user_meta": {"test": "test"}}`?

To make test closer to real-life scenario?

Reply:
This `meta` is `user_meta`.
See https://github.com/quiltdata/quilt/blob/16f25156608668a5495532372cf572f8de8206eb/api/python/quilt3/packages.py#L183-L185